### ISSUE TITLE
All earthquakes in Greece in 2018

### ISSUE THREAD

**Issue by nickospi** [jsoma/data-studio-projects#143]
# Pitch

I have found [this data set](http://www.geophysics.geol.uoa.gr/stations/maps/recent.html) from the Geophysics Department of the University of Athens, Greece. It contains geographical, chronological and measuring data of every earthquake that happened in Greece from 01.01.2018 until today.

## Details
Possible headline(s): Earthquakes in Greece in 2018

Data set(s): Earthquakes in Greece in 2018 with geographical, chronological and measuring data. 

Code repository: https://github.com/nickospi/data_studio/tree/master/code/01-Earthquakes_in_Greece_in2018 

Possible problems/fears/questions: NaN

## Work so far

A word on cleaning: I got this as a .tsv file straight from the above url, I opened it in pandas and got 55 columns which had 2 main issues: some of them were completely full of NaN values and had to be dropped, whereas some series from the original had split data in 2 columns [eg. on index 0, I got magnitude level in column 15 and on index 1 in column 16] so I had to merge these and drop NaNs]

### Graph 1: Which was the month with the most earthquakes in Greece in 2018 [up till today] ?
![allquakes](https://user-images.githubusercontent.com/39416389/42612887-d9a0c3c8-856b-11e8-923f-81b5c902850c.png)
### Graph 2: All 2018 quakes per magnitude and focal depth
![magvsdepth](https://user-images.githubusercontent.com/39416389/42612904-ee3143d0-856b-11e8-91af-f61fe65d00f5.png)
### Graph 3: All 2018 quakes in Greece on lattitude axis
![northvssouth](https://user-images.githubusercontent.com/39416389/42612895-e21e2810-856b-11e8-804c-3cc20f22f8fd.png)
### Graph 4. [wannabe Animated] All 2018 quakes per second
![astheyhappened](https://user-images.githubusercontent.com/39416389/42612912-f73329ee-856b-11e8-86ef-e8006c67d299.png)

Nice to have :
- a map showing earthquakes in Greece and focal depths/mag
- a graph with the actual names of the big cities with most quakes in a range (eg. 50 mi)
- Animate the fourth graph using date time data.
- would lile to work more on the legends [the warning in the 3rd one is about the legend]

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue issue has `[Project]` in the title
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far
- [x] I have received two comments of peer feedback on my pitch issue

**Comment by castorsia** [comment]
-Great work cleaning the data and visualising it in imaginative ways!
- A little work making the last 2 graphs easier to understand would make this a 10/10

**Comment by vpenney** [comment]
This is great. I really enjoy the different types of graphs that you use. I think it would be nice to invert that depth of earthquake graph (could it be as easy as changing the axis to sort by the opposite, i.e. ascending vs descending? That'd be nice!) The colors and fonts all look clean and polished as well. I'm excited to see your final project!

**Comment by malbasi** [comment]
I love the depth chart type and the overall idea. 

I'd like to see some analysis of the frequency and severity of the earthquakes. And the depth chart should be flipped upside down to convey its meaning better. The North Vs South graph is confusing. Maybe just axes labels or legend would help

**Comment by nickospi** [comment]
# Update
	
## Project content:
![allquakes](https://user-images.githubusercontent.com/39416389/42728036-567f49a4-877f-11e8-8658-fe17a63a408c.png)
![magvsdepth](https://user-images.githubusercontent.com/39416389/42728038-5fc8e7d6-877f-11e8-92f7-172152aa5a0f.png)
![northvssouth](https://user-images.githubusercontent.com/39416389/42728039-6200e206-877f-11e8-9183-992a289317e7.png)
![astheyhappened](https://user-images.githubusercontent.com/39416389/42728041-674651f6-877f-11e8-99eb-a1d75364ee1f.png)
  
## Changes
  
After the peer review, I put in some legends and changed the sorting in the y axis in the second graph. I guess that in an ideal world I could've also added a soil/grass vector as this axis actually depicts the focal depth of an earthquake providing that it wouldn't look amazingly kitsch.  

## Problems/Questions
  
- The dataset needed a lot of cleaning
- Can't figure how to actually animate the final graph

## Checklist
	
- [X ] I have included my visuals
- [ X] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [X ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by kellykiki** [comment]
Very nice work! What about finding a styling pattern to follow (mainly colors thing) for each chart of this project?

**Comment by ElinaMak** [comment]
Excellent work!!! The ONLY thing I would "reproach" as an editor is that those graphs are great for a savvy audience!

**Comment by jsoma** [comment]
* No vertical text on the y axis! I'd add magnitude next to the 5, and maybe add some explanatory text
* Your readers are NOT going to have a clue what's going on, and I don't think there's a real story here (is there?). What I'd do to make it a reasonable graphic is select a few points, highlight them, and add text to them to say "an earthquake happened at some point here, I was brushing my teeth, it was very scary" or something like that (but probably more interesting I hope)
* As time goes on I feel more and more like left-aligned titles are better than centered ones

**Comment by nickospi** [comment]
# Update
	
## Your project content: images/words/etc
	
![qks_1](https://user-images.githubusercontent.com/39416389/42977780-3054fd98-8b97-11e8-8b94-e105d3abbaa9.png)


![qks_2](https://user-images.githubusercontent.com/39416389/42977781-31fb1628-8b97-11e8-9939-273e11417a93.png)


![qks_3](https://user-images.githubusercontent.com/39416389/42977782-337523cc-8b97-11e8-98a9-a68bf8f5101c.png)


![qks_4](https://user-images.githubusercontent.com/39416389/42977784-3509184c-8b97-11e8-8ab7-50808774a685.png)

  
## Any changes in direction or topic?
  
Tried to make something newsworthy out of this dataset. I've tried to remake every chart on Adobe Illustrator.
	
## Problems/Questions
  
NaN	
## Checklist
	
- [X ] I have included my visuals
- [X] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [X] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by playfairbot** [comment]
Howdy! I'm a little robot, beep beep boop boop.

You need some feedback, let me summon @hakantan, @SiruiZhu, @zle2105 for you

**Comment by nickospi** [comment]
# Final

## Project visuals/text
All these make much more sense [here](https://nickospi.github.io/Earthquakes/) 

If you prefer the standalone version, it's this one:

![qks_1](https://user-images.githubusercontent.com/39416389/43028950-fd6f535a-8c50-11e8-839c-4bb6a341aaae.png)
![qks_2](https://user-images.githubusercontent.com/39416389/43028959-0b0f4ff6-8c51-11e8-91c9-6cc75eba4b80.png)
![qks_3](https://user-images.githubusercontent.com/39416389/43028962-0fcb7182-8c51-11e8-8bc9-57516e291158.png)
![qks_4](https://user-images.githubusercontent.com/39416389/43028969-179037fe-8c51-11e8-98f5-c777da9dc159.png)

## Details

**Headline:** 
What does it mean to be one of the world's most seismically active countries?
**Published website version:**
https://nickospi.github.io/Earthquakes/
**Code repository:**
https://github.com/nickospi/data_studio/tree/master/code/Earthquakes_in_Greece_in%202018/Finalised%20Project

**Final data set(s):** 
https://github.com/nickospi/data_studio/blob/master/code/Earthquakes_in_Greece_in%202018/Finalised%20Project/quakeswthdatetime.csv

## What did you find to be the most difficult part of this project?
The most difficult was definitely the cleaning of the dataset. Also making something newsworthy out of this, but it was not my prime goal to have something newsworthy.
## Are you satisfied with what you produced? Is there anything you would like to change or improve?
With my zero Adobe Illustrator skills and this dataset, I am happy. I feel that a story like that could be published on a let's say popular science website. 

## Checklist

- [X] I have included my visuals
- [X] I have posted my project to my project website
- [X] I have been updating my DIARY.md with details about my process
- [X] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by hakantan** [comment]
First things first. I really enjoy watching at those graphics. For the purpose of what we're doing – understanding the charting and finding visually appealing ways to do that – this is a 10/10. However, I am lost most of the times as to what it is that I can see in your charts.

For example:  The "Did you feel that"-chart is great, but what does 5.3R mean? (I would also make the bubbles smaller, because they are overlapping too often, I think). Only after I saw the other chart, I understood that this has to do with Richter scale. Because four charts a different in their layout I didn't think they belonged together. They do, tho. (In this case, I'd just re-explain what R stands for.

**Comment by SiruiZhu** [comment]
Wow, those visuals are very pretty!! and your black background with blue and red colors really nice-looking! 

I also have some problem when trying to understand your graphics:

- For the second-to-last graphic, the dots look so intensive that is hard to tell. Also the colors for 0.1R and 5.3R are quite similar, and hard to tell from the graphics. 

However it's still very attractive overall!

**Comment by zle2105** [comment]
These charts look amazing. I love the layout and color scheme. Since you have a ton of room and empty space, it might be nice to add some annotations or comments.
